lms/static/js/certificates/views/certificate_whitelist_editor.js

```diff
@@ -82,9 +85,9 @@
                     return re.test(email);
                 },
 
-                showMessage: function(message){
+                escapeAndShowMessage: function(message){
                     $(this.message_div +  ">p" ).remove();
-                    this.$(this.message_div).removeClass('hidden').append("<p>"+ gettext(message) + "</p>");
+                    this.$(this.message_div).removeClass('hidden').append("<p>"+ _.escape(message) + "</p>");
                 },
 
                 showSuccess: function(caller, add_model, message){
```

Comment:
I'm not proposing you rename this, because there are helpers coming that may one day change the recommendation for this page.  However, you may want to verify that there are no outside callers of showMessage() or showError() that might be including html in the message.